wordpress/wp-content/themes/headless/acf-json/group_5f4f3a7bdc981.json

```diff
@@ -58,5 +58,9 @@
     "hide_on_screen": "",
     "active": true,
     "description": "",
-    "modified": 1599028053
+    "show_in_graphql": 0,
+    "graphql_field_name": "flexBlockquote",
```

Comment:
small thing, but this one doesn't need a graphql field name (the others are empty strings)